lib/DDGC/GitHub.pm

```diff
@@ -289,6 +291,10 @@ sub update_repo_pulls {
     push @gh_pulls, $self->update_repo_pull_from_data($gh_repo, $_)
         for @$pulls_data;
 
+    my @gh_events;
+    push @gh_events, $self->update_repo_pull_from_data($gh_repo, $_, 'github_pull')
```

Comment:
You're right! In my effort to wrap this up quickly, I wasn't careful with my copy/paste.
Thanks :+1: